ui/js/group.js

```diff
@@ -160,29 +160,40 @@ require(['jquery', 'oae.core'], function($, oae) {
 
         // Structure that will be used to construct the left hand navigation pages
         var lhNavPages = [];
-        // Only show the recent activity to group members
-        if (groupProfile.isMember) {
-            lhNavPages.push({
-                'id': 'activity',
-                'title': oae.api.i18n.translate('__MSG__RECENT_ACTIVITY__'),
-                'icon': 'fa-tachometer',
-                'closeNav': true,
-                'layout': [
-                    {
-                        'width': 'col-md-12',
-                        'widgets': [
-                            {
-                                'name': 'activity',
-                                'settings': {
-                                    'context': groupProfile,
-                                    'canManage': groupProfile.isManager
-                                }
+
+        // Show the activity and about widgets on the main page
+        lhNavPages.push({
+            'id': 'activity',
+            'title': oae.api.i18n.translate('__MSG__RECENT_ACTIVITY__'),
+            'icon': 'fa-tachometer',
+            'closeNav': true,
+            'layout': [
+                {
```

Comment:
On `sm` and `xs` resolutions, the right hand bar should be hidden. It should not jump below the activity widget.

![screen shot 2015-04-20 at 19 51 58](https://cloud.githubusercontent.com/assets/109850/7244864/ce4bb170-e796-11e4-94c9-0728a9668b94.png)